cli/src/commands/describe.rs

```diff
@@ -67,6 +68,15 @@ pub(crate) struct DescribeArgs {
         conflicts_with = "stdin"
     )]
     message_paragraphs: Vec<String>,
+    /// Add a footer entry
+    ///
+    /// The value is a commit template. The output generated with that template
+    /// must be a valid footer entry.
+    ///
+    /// If multiple revisions are specified, the same footer line will be used
+    /// for all of them.
```

Comment:
clarify that that's only true if the template doesn't depend on the commit (right?)